src/Essentials/src/SecureStorage/SecureStorage.shared.cs

```diff
@@ -27,15 +40,27 @@ public static Task SetAsync(string key, string value)
 			if (value == null)
 				throw new ArgumentNullException(nameof(value));
```

Comment:
Can we move this logic into each platform? This way the static API is very very thin.

Reply:
This is better be done after reaching API stability.